kinto/core/logs.py

```diff
@@ -10,6 +10,19 @@
 logger = structlog.get_logger()
 
 
+def reset_logger():
+    """Hack to work around https://github.com/hynek/structlog/issues/71.
+
+    This clears a magic field in the logger so that it will regenerate
+    the right kind of logger on its next use.
+
+    Do this when you call structlog.configure(). Otherwise, some parts
```

Comment:
Maybe we could provide our own `configure` function that includes `logger._logger = None` because I don't see a use case in kinto for using `reset_logger` besides configuring the logger.